multicast/exceptions.py

```
@@ -196,12 +196,12 @@
 	from . import argparse  # skipcq: PYL-C0414
 	from . import logging  # skipcq: PYL-C0414
 	import functools
-except Exception as err:
-	baton = ImportError(err, str("[CWE-758] Module failed completely."))
+except ImportError as _cause:
+	baton = ImportError(_cause, str("[CWE-758] Module failed completely."))
```

Ruff (0.8.2)

Unnecessary `str` call (rewrite as a literal)

Replace with string literal

(UP018)